### PR TITLE
Task/multitarget sdk

### PR DESCRIPTION
**Pull request by KevReed**
build working for both platforms from VS2019 and via dotnet build

### CODE REVIEW

File: src/DSharp.Sdk/Sdk/Sdk.targets [hunk 1]
```diff
@@ -1,53 +1,12 @@
-<Project>
-
+<Project TreatAsLocalProperty="TaskFolder">
+  
   <PropertyGroup>
+    <TaskFolder Condition=" '$(MSBuildRuntimeType)' == 'Core' ">netstandard2.0</TaskFolder>
+    <TaskFolder Condition=" '$(MSBuildRuntimeType)' != 'Core' ">net461</TaskFolder>
```
Comment (fred-perkins) [inline src/DSharp.Sdk/Sdk/Sdk.targets]: Still says net461

File: src/DSharp.Web.Sdk/DSharp.Web.Sdk.csproj [hunk 2]
```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(SolutionDir)NuGetCommon.props" Condition="Exists('$(SolutionDir)NuGetCommon.props')" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```
Comment (fred-perkins) [inline src/DSharp.Web.Sdk/DSharp.Web.Sdk.csproj]: shouldn't this be net471 as with the other target file?

File: src/NuGetCommon.props [hunk 3]
```diff
@@ -21,9 +21,14 @@
 
     </PropertyGroup>
 
-    <XmlPoke XmlInputPath="Sdk/Sdk.props" 
-             Query="//ItemGroup//PackageReference[@Include='DSharp.Mscorlib']/@Version" 
+    <ItemGroup>
+      <VersionSyncedImports Include="DSharp.Mscorlib"/>
```
Comment (fred-perkins) [inline src/NuGetCommon.props]: What does this do?

Comment (KevReed) [reply src/NuGetCommon.props]: updates the version numbers for package references listed by the VersionSyncedImports ItemGroup, previously the reference to DSharp.Mscorlib was hardcoded in the poke task

File: src/DSharp.Sdk/Sdk/Sdk.targets (hunk 1 above)
Comment (KevReed) [reply src/DSharp.Sdk/Sdk/Sdk.targets]: This condition is to locate the build tasks depending on whether we are running from msbuild (net framework) or dotnet build (netstandard). All DSharp assemblies are still built against 461

File: src/DSharp.Web.Sdk/DSharp.Web.Sdk.csproj (hunk 2 above)
Comment (KevReed) [reply src/DSharp.Web.Sdk/DSharp.Web.Sdk.csproj]: i kept the sdk projects inline with the rest of the build tooling

**Pull request merged**